cypress/integration/questions.ts

```diff
@@ -0,0 +1,65 @@
+import faker from 'faker';
+
+context('Questions tests', () => {
+  before(() => {
+    cy.resetDB();
+  });
+
+  beforeEach(() => {
+    cy.viewport(1100, 1100);
+    cy.visit('/');
+  });
+
+  const textQuestion = faker.lorem.words(2);
+
+  it('User officer search questions', () => {
+    cy.login('officer');
+
+    cy.navigateToTemplatesSubmenu('Proposal templates');
+
+    cy.contains('default template')
+      .parent()
+      .get("[title='Edit']")
+      .first()
+      .click();
+
+    cy.get('[data-cy=show-more-button]').last().click();
```

Comment:
Maybe we should avoid using `.last()` or `.first()` as much as possible. It is easier to maintain it later if it is better targeted.